javascript-functions/main.js

```diff
@@ -0,0 +1,39 @@
+function converMinutesToSeconds(minutes) {
+  var totalSeconds = minutes * 60;
+  return totalSeconds;
+}
+var converMinutesToSecondsResult = converMinutesToSeconds(3);
+console.log('converMinutesToSecondsResult', converMinutesToSecondsResult);
+
+function greet(name) {
+  var greeting = 'Hey, ' + name;
+  return greeting;
+}
+
+var greetName = greet('Cooper');
+console.log('greetName:', greetName);
+
+function getArea(width, height) {
+  var totalArea = width * height;
+  return totalArea;
+}
+
+var getAreaResult = getArea(10, 6);
+console.log('getAreaResult:', getAreaResult);
+
+function getFirstName(person) {
+  var firstName = person;
+  return firstName;
+}
```

Comment:
You should be getting the value of the `firstName` property on the `person` object